### DISCUSSION TITLE
Pay one time for multiple invoice

### DISCUSSION THREAD

**Discussion by boudo2017** [Blair2004/NexoPOS#519] [Ideas]
Hi,
It is necessary to add this feature:"
Pay one time for multiple invoice of the unique customer (monthly or periodic payments )."
regards

**Comment by Blair2004** [comment]
Are you talking about subscription (recurring orders) ?

**Reply by boudo2017** [reply]
No,
A customer made 2 orders today 
3 tomorrow 1 one day after tomorrow 14 orders next week 10 orders the third week and 3 orders the fourth week, the second month made 7 orders as total.
Now, he made 40 unpaid orders with total due amount 3250 $
two ways to pay :
1- track all orders and update it or add payment for it, one by one(not recommended and refused by all customers ).
2-add a "pay" button for each customer , one click -> all orders are paid ,no due amounts .
( this feature used by ultimatepos -laraval environment)

**Reply by Blair2004** [reply]
Every payments needs to have a payment type. I'm rather thinking this should be a module.

**Comment by boudo2017** [comment]
you can customize pay button to do that.(you can use multiple payment type for a single bill too)
 You can add payments type note ,payment method type as you need.
cash by defaults,(card ,deposit) as options .
we are talking about a real situation, we have that all time, customer while paying total or partial due amount didn't ask to pay a specific bill but a specific amount.
